### PR TITLE
Update Notif Section 

### LINKED ISSUE #147: Remove `Get Notified` Push Notification
As you know we are using the Push Notification technique to send motifs to users when a new episode is live.
Until now we have more than 150 devices registered, but the problem is that conversion rate is very low as Push Notification works well only for websites with daily users in most of the cases (not the case for geeksblabla, a new episode every week ). 

That's why I suggest updating the Notification section to a simple text when we recommend the user to subscribe to the Youtube channel (Fb pages ) and activate Notification. 
we can also provide a simple form to register to the geeksblabla  email list using http://tinyletter.com/.    

@aboullaite @ismailElazizi @ezzarghili @iMeriem   ?

### PR DESCRIPTION
**Pull request by yjose**
* Remove OneSignal implementation.
* Facebook and youtube channels instead 
* fix #147

### CODE REVIEW

**Comment by mouradaouinat** [comment]
I think there's a problem with cache, the template is still expecting the OneSignal component

**Comment by yjose** [comment]
@mouradaouinat  fixed

File: src/components/Home/Notification/index.js [hunk 1]
```diff
@@ -15,10 +18,47 @@ export default () => {
...
+          </a>
         </div>
+        <p>
+          Or You can subscribe to our{" "}
+          <a
+            href="https://tinyletter.com/geeksBlabla"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Email List
```
Comment (ezzarghili) [inline src/components/Home/Notification/index.js:58]: I the term Mailing List is more common?

File: src/components/Home/Notification/index.js [hunk 2]
```diff
@@ -15,10 +18,47 @@ export default () => {
       {dark ? <Notif className="notif" /> : <NotifLight className="notif" />}
       <div className="content">
         <h1> Get Notified </h1>
-        <p> Get instant updates about GeeksBlaBla </p>
-        <div className="actions">
-          <GetNotification />
+        <p>
+          The best way to get notified about live streams is by subscribing to
```
Comment (ezzarghili) [inline src/components/Home/Notification/index.js:22]: Simpler wording: The to get live streams notifications subscribe to our Youtube Channel and Facebook Page (Make sure to enable notifications 😉)

File: src/components/Home/Notification/index.js [hunk 3]
```diff
@@ -15,10 +18,47 @@ export default () => {
...
+          <a
+            href="http://geeksblabla.com/rss.xml"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <Rss height={30} alt="GeeksBlabla RSS Feed " />
+          </a>
         </div>
+        <p>
+          Or You can subscribe to our{" "}
```
Comment (ezzarghili) [inline src/components/Home/Notification/index.js:52]: Or *you* can subscribe to our{" "}

**Comment by ezzarghili** [comment]
LGTM, I commented for some smalls changes wrt to wording.

**Pull request merged**